UnityProject/Assets/IngameDebugConsole/Scripts/DebugLogUnitystationCommands.cs

```diff
@@ -764,5 +764,19 @@ private static void GenerateCyborg()
 			}
 
 		}
+
+		[ConsoleMethod("reset-movement", "Resets all movement values. Helpful if you get stuck for no reason.")]
+		public static void ResetMovementStats()
+		{
+			if(IsAdmin() == false) return;
```

Comment:
PR description says you don't need an admin

Reply:
oh shoot, you're right.

Reply:
fixed.